### PR TITLE
Add disclaimer

### PR DESCRIPTION
**Pull request by ranjan-stha**
Responded to issues #2 and #3 
* Add the disclaimer
* Handle the local storage for caching a piece of data
* Update the streamlit version.

### CODE REVIEW

File: frontend_src/app.py [hunk 1]
```diff
@@ -54,227 +56,234 @@
...
-
-    st.session_state["inform_severity_data_path"] = os.path.join(
-        st.session_state["tabular_data_data_path"],
-        "acaps_inform_severity",
-        "INFORM Severity latest.xlsx",
-    )
-    st.session_state["inform_severity_df"] = _load_information_severity_index_data()
+local_storage = LocalStorage()
+
+cached_disclaimer_data = local_storage.getItem("cpaor_data")
```
Comment (AdityaKhatri) [inline frontend_src/app.py:61]:
```suggestion
cached_disclaimer_data = local_storage.getItem("cpaor_consent_confirm")
```

File: frontend_src/app.py [hunk 2]
```diff
@@ -54,227 +56,234 @@
...
+if not cached_disclaimer_data:
+    local_storage.setItem("cpaor_data", "disclaimer_displayed")
```
Comment (AdityaKhatri) [inline frontend_src/app.py:63]:
```suggestion
    local_storage.setItem("cpaor_consent_confirm", "false")
```

**Review by AdityaKhatri — CHANGES_REQUESTED** [review]

File: frontend_src/app.py (hunk 1 above)
Comment (ranjan-stha) [reply frontend_src/app.py]: done

File: frontend_src/app.py (hunk 2 above)
Comment (ranjan-stha) [reply frontend_src/app.py]: done

**Review by AdityaKhatri — APPROVED** [review]

**Pull request merged**